src/pages/settings/Wallet/WalletPage/BaseWalletPage.js

```diff
@@ -483,9 +483,9 @@ function BasePaymentsPage(props) {
     );
 }
 
-BasePaymentsPage.propTypes = propTypes;
-BasePaymentsPage.defaultProps = defaultProps;
-BasePaymentsPage.displayName = BasePaymentsPage;
+BaseWalletPage.propTypes = propTypes;
+BaseWalletPage.defaultProps = defaultProps;
+BaseWalletPage.displayName = BaseWalletPage;
```

Comment:
There was typo here. It should be `'BaseWalletPage'`. And later it caused [this regression](https://github.com/Expensify/App/issues/28057).